app/service.js

```diff
@@ -42,7 +42,9 @@ function getDocumentMetadata(url, window) {
     width: 500,
   }];
 
-  console.log(`Generated Metadata for ${url}:\n${JSON.stringify(metadata)}`); // eslint-disable-line no-console
+  if (process.env.NODE_ENV !== 'test') {
```

Comment:
Alternatively, if you want to see these locally, but not on Circle-CI, we can probably change this to:

```js
if (!process.env.CI) {
   console.log(`...`);
}
```
(per https://circleci.com/docs/environment-variables/)